foreshadow/transformers/transformers.py

```diff
@@ -22,32 +40,48 @@ def _get_modules(classes, _globals, mname):
 
     Args:
         classes: A list of classes
-        _globals: The globals in the callee's context
+        globals_: The globals in the callee's context
         mname: The module name
+        wrap: True to wrap modules, False to not (manual decorator
```

Comment:
Where would we use the wrap switch statement?

Reply:
I can remove it